extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -147,10 +151,16 @@ export class LiveListManager {
   fetchDocument_() {
     let url = this.url_;
     if (this.latestUpdateTime_ > 0) {
-      url = addParamToUrl(
+      url = addParamsToUrl(
         url,
-        'amp_latest_update_time',
-        String(this.latestUpdateTime_)
+        /** @type {JsonObject} */ ({
+          'amp_latest_update_time': String(this.latestUpdateTime_),
+          // AMP Caches do not always evict entries from their caches.
+          // A random number from the max safe range without BigInts helps add confidence the document is fresh.
+          'amp_random': String(
+            Math.floor(Math.random() * AMP_LIVE_LIST_MAX_RANDOM_NUMBER)
```

Comment:
`Date.now` is more deterministic, and will allow the cache to serve the same response (via their request time) to multiple people.

Reply:
@Gregable requested this to be a random number, not based on the local clock of the device.

If a device has a clock set to the near future, it could create stale or inconsistent results on the server cache.

Am happy to use either a Date/Time Stamp or random number, or combination thereof for this short term workaround.